__tests__/1sever.test.js

```diff
@@ -66,24 +66,24 @@ describe("GET /api/reviews/3", ()=>{
           })
       })
   })
-  it("should return an review object from given ID with correct properties",()=>{
-    return request(app).get('/api/reviews/4').expect(200).then((response)=>{
-        response.body.review.forEach((reviews)=>{
-          
-          expect(reviews.review_id).toBe(4);
 
-        })
-    })
-})
 
 it("should return Error msg if input ID is not a number",()=>{
   return request(app).get('/api/reviews/r').expect(400).then((response)=>{
     expect(response.body.msg).toBe('Invalid ID');
-      
+     
 
       })
   })
 
+  it.only("should return Error msg if input ID is not Found",()=>{
+    return request(app).get('/api/reviews/100000').expect(404).then((response)=>{
+      expect(response.body.msg).toBe('Not Found');
+        
+  
+        })
+    })
+
```

Comment:
You also need to check for an invalid ID (for example the ID is a string) which you should be expecting a 400 error for 👀